git_machete/cmd.py

```diff
@@ -1684,6 +1700,7 @@ def usage(c=None):
 
             Options:
             -l, --list-commits            Additionally lists the messages of commits introduced on each branch.
+            --color[=WHEN]                Colorize the output; WHEN can be 'always' (default if omitted), 'auto', or 'never';
```

Comment:
I'd say `auto` should be the default

Reply:
Aaaah ok it's a bit confusing:
* `auto` is the "default" default if `--color` isn't specified BUT
* `always` is the default value for `--color` flag
Pls think how to clarify that... without writing a whole essay here ;)

Reply:
Aaaah ok it's a bit confusing:

* `auto` is the "default" default if` --color` isn't specified BUT
* `always` is the default value for `--color` flag

Pls think how to clarify that... without writing a whole essay here ;)

Reply:
Btw consider making the argument non-optional as a workaround, for the sake of clarity